crowd_anki/github/github_importer.py

```diff
@@ -1,18 +1,14 @@
-import tempfile
-import zipfile
-from urllib.error import HTTPError, URLError
-from urllib.request import urlopen
-
-from io import BytesIO
 from pathlib import Path
-
-import aqt.utils
+import sys
 from aqt import QInputDialog
+
+from dulwich import porcelain
+from dulwich.errors import NotGitRepository
 from ..importer.anki_importer import AnkiJsonImporter
-from ..utils import utils
 
 BRANCH_NAME = "master"
-GITHUB_LINK = "https://github.com/{}/archive/" + BRANCH_NAME + ".zip"
+GITHUB_ZIP_URL = "https://github.com/{}/archive/" + BRANCH_NAME + ".zip"
+GITHUB_REPO_URL = "https://github.com/{}.git"
 
 
 class GithubImporter(object):
```

Comment:
should we rename this to GitImporter and accept just the repo URL?

Reply:
I'd say sure.

Reply:
Done